exercises/lens-person/description.md

```diff
@@ -2,8 +2,12 @@
 
 Use lenses to update nested records (specific to languages with immutable data).
 
-Updating fields of nested records is kind of annoying in Haskell. One solution
-is to use [lenses](https://wiki.haskell.org/Lens).  Implement several record
-accessing functions using lenses, you may use any library you want. The test
-suite also allows you to avoid lenses altogether so you can experiment with
-different approaches.
+Updating fields of nested, immutable records is kind of annoying.
+The code for such cases is as cumbersome as the structure is deep.
+If you have, say, a Person, that contains an Address, which has a Street, that has a Number, updating the Number requires creating a new Street with the new Number, then a new Address with the new Street and, finally, a new Person with the new Address.
+Confused already?
+
+One solution to this problem is to use [lenses](https://en.wikibooks.org/wiki/Haskell/Lenses_and_functional_references).
+
+Implement several record accessing functions using lenses.
+The test suite also allows you to avoid lenses altogether so you can experiment with different approaches.
```

Comment:
Hmm, the _Haskell_ test suite might allow that, but this is funny thing to say for _all_ tracks. I'd remove that last sentence.

Reply:
This last sentence was already there and I just reformatted. Maybe it would make sense to leave it for now, and have someone else who can verify if this is true or not remove it?

Reply:
Even better, open an issue describing this and let the conversation happen and the change derive from that issue, while accepting the format change here?